src/material-table.js

```diff
@@ -438,8 +438,6 @@ export default class MaterialTable extends React.Component {
           if (row.tableData.checked) {
             selectedRows.push(row);
           }
-
-          row.tableData.childRows && findSelecteds(row.tableData.childRows);
```

Comment:
Are you sure removing this line? Because if data has treefied it will find selected data as nestedly.